e2e/tshark/tshark_test.go

```diff
@@ -41,58 +40,42 @@ func TestTshark(t *testing.T) {
 	target, err := kn.NewInstance("busybox")
 	require.NoError(t, err, "error creating instance")
 
-	err = target.SetImage(ctx, "busybox")
-	require.NoError(t, err, "error setting image")
-
-	err = target.SetCommand("sleep", "infinity")
-	require.NoError(t, err, "error setting command")
+	require.NoError(t, target.SetImage(ctx, "busybox"))
+	require.NoError(t, target.SetCommand("sleep", "infinity"))
 
 	t.Log("deploying minio as s3 backend")
-	err = kn.MinioClient.DeployMinio(ctx)
-	require.NoError(t, err, "error deploying minio")
+	require.NoError(t, kn.MinioClient.DeployMinio(ctx))
 
 	t.Log("getting minio configs")
 	minioConf, err := kn.MinioClient.GetConfigs(ctx)
 	require.NoError(t, err, "error getting S3 (minio) configs")
 
+	keyPrefix := "tshark/" + scope
```

Comment:
We have `obsy.New()` and `bittwister.New()`. Why we don't have `tshark.New()` here?

Reply:
Because they have some initilization like assigning default values, but tshark does not have. We can have it for sake of consistency though

Reply:
oh, I just saw that the tShark struct has too many params and does not really need an initialization. see:

```go
type Tshark struct {
	Image    string
	VolumeSize resource.Quantity
	S3AccessKey string
	S3SecretKey string
	S3Region string
	S3Bucket string
	CreateBucket bool
	S3KeyPrefix string
	S3Endpoint string
	UploadInterval time.Duration
}
```

so if we use a New func, it will need to pass an option which has the whole fields and it will be just redundant. So I suggest to keep it as it is right now.